### PR TITLE
[review and merge in aug 2024] remove fal related resources

### PR DESCRIPTION
**Pull request by kiwamizamurai**
**Update**
We will review and merge this PR around Aug 2024 to give users time to adjust to this adapter sunset. to ease the removal , we're going to:

- phase 1: add a callout that the dbt fal adapter isn't being maintained/supported anymore. this helps set expectations for users.
- phase 2: fully merge THIS PR to remove the docs around aug 2024 time.

## What are you changing in this pull request and why?

Delete dbt-fal documentation because it has been archived

https://github.com/fal-ai/dbt-fal 

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [x] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [x] Add an entry for deleted pages in `website/vercel.json`
- [x] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

**Comment by kiwamizamurai** [comment]
@mirnawong1

could you review this pr?

**Comment by mirnawong1** [comment]
hey @kiwamizamurai thanks so much for your proactive pr - really appreciate it ✨!  we've decided to approach this sunset in phases since some users are still using `dbt-fal` and we want to ease the removal a bit: 

- phase 1: add a callout that the dbt fal adapter isn't being maintained/supported anymore. this helps set expectations for users.
- phase 2: fully remove the docs in a couple of months time using your PR. 

so we'll wait a couple of months and then use your pr to completely remove the docs from the docs site. I've also checked with the adapter maintainer on this @chamini2 to confirm too! let me know if you have other ideas/opinions but this pr is really going to help users when we merge it in a couple of months!

**Comment by mirnawong1** [comment]
converting this to draft until aug 2024

**Comment by mirnawong1** [comment]
phase 1 pr here: https://github.com/dbt-labs/docs.getdbt.com/pull/5454

**Comment by kiwamizamurai** [comment]
@mirnawong1 
Thank you very much.
https://github.com/dbt-labs/docs.getdbt.com/pull/5454

OK, I wait for the phase 2

**Comment by mirnawong1** [comment]
confirming if ok to remove from teh site now. will change pr status to 'ready for review' soon once confirmed.

**Comment by mirnawong1** [comment]
hey @kiwamizamurai ! happy thursday and hope you're doing well! i'm trying to resolve the merge conflicts but unfortunately i don't have permission to your forked. any chance you're able to?

**Comment by kiwamizamurai** [comment]
@mirnawong1 

ok, i'll do it

File: website/vercel.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "trailingSlash": false,
   "redirects": [
     {
-      "source":  "docs/cloud/secure/environment-permissions",
-      "destination":  "docs/cloud/manage-access/environment-permissions",
+      "source": "docs/cloud/secure/environment-permissions",
+      "destination": "docs/cloud/manage-access/environment-permissions",
```
Comment (kiwamizamurai) [inline website/vercel.json]:
```suggestion
      "source":  "docs/cloud/secure/environment-permissions",
      "destination":  "docs/cloud/manage-access/environment-permissions",
```

File: website/vercel.json [hunk 2]
```diff
@@ -1054,7 +1054,6 @@
       "destination": "/docs/get-started/develop-in-the-cloud",
       "permanent": true
     },
```
Comment (kiwamizamurai) [inline website/vercel.json]:
```suggestion
    },

```

**Comment by kiwamizamurai** [comment]
@mirnawong1 
please check it

**Comment by mirnawong1** [comment]
hey @kiwamizamurai , that's awesome and thanks so much for your contribution! I've messaged the maintainers and will get this merged for you!

**Review by mirnawong1 — APPROVED** [review]
lgtm! thanks @kiwamizamurai for opening this up again and really appreciate this great contribution!

**Pull request merged**